kogitoq/kogito-common/src/main/java/life/genny/kogito/common/service/KogitoService.java

```diff
@@ -1,71 +1,89 @@
 package life.genny.kogito.common.service;
 
-import java.io.IOException;
-import java.lang.invoke.MethodHandles;
-import java.net.URI;
-import java.net.http.HttpClient;
-import java.net.http.HttpRequest;
-import java.net.http.HttpResponse;
-import java.util.concurrent.TimeUnit;
-
 import javax.enterprise.context.ApplicationScoped;
 import javax.inject.Inject;
 import javax.json.bind.Jsonb;
 import javax.json.bind.JsonbBuilder;
-import javax.ws.rs.core.Response;
+import javax.persistence.EntityManager;
 
 import org.jboss.logging.Logger;
 
+import life.genny.kogito.common.core.Dispatch;
+import life.genny.kogito.common.core.ProcessAnswers;
+import life.genny.kogito.common.utils.KogitoUtils;
+import life.genny.qwandaq.managers.capabilities.role.RoleManager;
 import life.genny.qwandaq.models.ServiceToken;
+import life.genny.qwandaq.models.UserToken;
+import life.genny.qwandaq.utils.BaseEntityUtils;
+import life.genny.qwandaq.utils.DatabaseUtils;
+import life.genny.qwandaq.utils.DefUtils;
+import life.genny.qwandaq.utils.FilterUtils;
+import life.genny.qwandaq.utils.QwandaUtils;
+import life.genny.qwandaq.utils.SearchUtils;
+import life.genny.serviceq.intf.GennyScopeInit;
 
-@ApplicationScoped
-public class KogitoService {
 
-	private static final Logger log = Logger.getLogger(MethodHandles.lookup().lookupClass());
+/**
+ * Class to hold bare essentials for bringing up classes that contain service task methods for Kogito Workflows
+ */
+@ApplicationScoped
+public abstract class KogitoService {
```

Comment:
Repurpose kogitoservice instead of deleting it

Reply:
Great use of existing class